### PR TITLE
Fix tenant status filter

### LINKED ISSUE #11974: Invalid filter used in the Tenants list
### Describe the bug
When clicking on the State badge on the `/admin/Tenants` list, the data does not get filtered as it should. The problem is that the filter in the generated URL has this `Options.Filter=Uninitialized` instead of `Options.Status=Uninitialized`. Also, the URL should have `&` when `?` is part of the URL and `?` if it does not exists in the URL.

image

### To Reproduce
1. Go to Tenants list `/admin/Tenants`
2. Next to any tenant click on the badge that shows the tenant's status
3. You'll notices that the list did not get filtered

### Expected behavior
When clicking on "Uninitialized" badge, only tenants with "Uninitialized" should show up.

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #11974

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml [hunk 1]
```diff
@@ -123,7 +125,7 @@
                                 {
                                     <span class="badge ta-badge font-weight-normal"><i class="fa fa-image text-info" aria-hidden="true"></i> @entry.ShellSettings["RecipeName"]</span>
                                 }
-                                <a href="@FullRequestPath&Options.Filter=@entry.ShellSettings.State">
+                                <a href="@(requestPath)Options.Status=@entry.ShellSettings.State">
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml:128]:
```suggestion
                                <a asp-controller="Admin" asp-action="Index" asp-route-Options.Status="@entry.ShellSettings.State">
```

File: src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml [hunk 2]
```diff
@@ -100,6 +100,8 @@
         </li>
         @if (Model.ShellSettingsEntries.Count > 0)
         {
+            var requestPath = FullRequestPath + (FullRequestPath.Contains('?') ? "&": "?");
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml:103]: Remove

```suggestion
            
```

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml:103]: Believe or not why I didn't think about this ;) may be someone used `FullRequestPath` for a reason

File: src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml [hunk 3]
```diff
@@ -100,6 +100,7 @@
         </li>
         @if (Model.ShellSettingsEntries.Count > 0)
         {
+
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml:103]: remove extra line

**Review by Skrypt — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml (hunk 3 above)
Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml]: I remove it before I saw your comment ;)

**Pull request merged**